io.github.zer0x.qurancompanion.yml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.zer0x.qurancompanion
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.kde.Sdk.Locale
+command: quran-companion
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it support wayland then line 11 should be `fallback-x11` otherwise line 12 should be removed.

Reply:
done.